lib/pure/strutils.nim

```diff
@@ -112,10 +106,18 @@ const
   Newlines* = {'\r', '\n'}
     ## The set of characters a newline terminator can consist of
 
-  # same as {'!', '\"', '#', '$', '%', '&', '\'', '(', ')', '*', '+', ',', '-', '.', '/', ':', ';', '<', '=', '>', '?', '@', '[', '\\', ']', '^', '_', '`', '{', '|', '}', '~'}
-  # same as {'!'..'/', ':'..'@', '['..'`', '{'..'~'}
-  Punctuation* = {'!'..'~'} - Letters - Digits
+  ControlChars* = {'\0'..'\31', '\127'}
+    ## The set of all ASCII control characters.
```

Comment:
how about:
```nim
    ## The set of all ASCII control characters, i.e. that does not occupy
    ## a printing position on a display
    ## (see also https://www.cplusplus.com/reference/cctype/iscntrl/)
```

likewise, PrintableChars doc comment could similarly be expanded, eg:
```nim
    ## The set of all ASCII printable characters, i.e. that occupy a printing
    ## position on a display (see also https://www.cplusplus.com/reference/cctype/isprint/)
```